### ISSUE TITLE
Killed 9

### ISSUE THREAD

**Issue by jecampagne** [LSSTDESC/tomo_challenge#16]
Hi Folks
I have tried for fun a GaussianProcess classifier as I have done for Gradient Boosting. But the program crash
Executing:  myGaussianProcess riz {'bins': 6, 'colors': True, 'errors': False}
Initializing classifier...
Training...
Finding bins for training data
Fitting classifier
Killed: 9

The only thing  I have to say is that it crashes after a while so I guess that it is fitting but then...crash :(

**Comment by joezuntz** [comment]
Hi @jecampagne - that's usually a memory error I think - is it possible your code was trying to allocate too much RAM?

**Comment by jecampagne** [comment]
Well I will have a look but this would be strange. Currently I am running an other Classiifier and python is using 4.21Go which certainly correspond more or less to the amount of training set and I have 8Go RAM. So, I do not expect a crash due to too RAM. But I will x-check.

**Comment by jecampagne** [comment]
If fact @joezuntz  you are perfectly wright
The RAM blew up using Gaussian Process default
I do not know if it is related to the scikit-learn implementation or the use in our challenge
![image](https://user-images.githubusercontent.com/20539759/85206782-951b2480-b324-11ea-9de4-d6ae1d4797d9.png)

**Comment by joezuntz** [comment]
You could try this on NERSC maybe where there should be plenty of RAM?

**Comment by EiffL** [comment]
I think this is solved, the answer was "this requires a big computer ^^' "
